test/test_type_promotion.py

```diff
@@ -172,18 +172,19 @@ def test_mixed_type_backward(self, device):
                 func(x, y).sum().backward()
 
     def _get_test_tensor(self, device, dtype, remove_zeros=False):
-        shape = [20, 20, 20]
+        shape = [10, 10, 10]
```

Comment:
Why did this change?

Reply:
I could change it back, but it just seemed excessive and we have a lot of tests to run.

Reply:
You don't need to change it back, I was just curious